core/include/tee_bench_core.h

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2016, Linaro Limited
+ *
+ * This software is licensed under the terms of the GNU General Public
```

Comment:
release this under the 2 clause BSD-like license.
you are still allowed to release it under a GPL like license in the scope of a linux driver/application.